lib/common/tests/rules/pcmk__evaluate_attr_expression_test.c

```diff
@@ -673,6 +686,19 @@ op_eq_fails(void **state)
     assert_attr_expression(EXPR_OP_EQ_FAILS, pcmk_rc_op_unsatisfied);
 }
 
+#define EXPR_UNDEFINED_EQ_EMPTY_FAILS                   \
+        "<" PCMK_XE_EXPRESSION " " PCMK_XA_ID "='e' "   \
+        PCMK_XA_ATTRIBUTE "='boo' "                     \
+        PCMK_XA_OPERATION "='" PCMK_VALUE_EQ "' "       \
+        PCMK_XA_VALUE "='' />"
+
+static void
+op_undefined_eq_empty_fails(void **state)
```

Comment:
ditto, a bit more counterintuitive here in that the name looks like the op is `not_defined` (we have other `op_undefined` tests)